components/bio-formats-tools/src/loci/formats/tools/ImageInfo.java

```diff
@@ -989,6 +994,15 @@ public boolean testRead(String[] args)
     throws FormatException, ServiceException, IOException
   {
     DebugTools.enableLogging("INFO");
+
+    if (args.length == 1 && HELP_ARGUMENTS.contains(args[0])) {
```

Comment:
This PR could be rewritten to accept `-h` among a bunch of other arguments, maybe set a state variable. I'd want a bit more guidance, though: should then `parseArgs` return `true`? Should the command go ahead even if usage is also printed if the other arguments are valid? The current approach hopes to avoid such questions, unless we anticipate introducing more context-specific help later (as with `bin/omero foo -h`).